<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eth_test</class>
 <widget class="QWidget" name="Eth_test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eth_test</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox{
background-color : white;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">

border : 2px solid;
border-color : black;
background-color : white; border-radius: 20px</string>
    </property>
    <property name="text">
     <string>EtherNet_TEST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="Network_group">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>221</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Agency FB&quot;;

border : 2px solid;
border-radius : 25px;</string>
    </property>
    <property name="title">
     <string>Network Setting</string>
    </property>
    <widget class="QGroupBox" name="IP_BOX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>IP Address</string>
     </property>
     <widget class="QLineEdit" name="IP_LE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.10.123</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IP_Set">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>25</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Net_Mask_Box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Net Mask</string>
     </property>
     <widget class="QPushButton" name="NM_Set">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NM_LE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>255.255.255.0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GW_Group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Gateway</string>
     </property>
     <widget class="QPushButton" name="GW_Set">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="GW_LE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>531</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Agency FB&quot;;

border : 2px solid;
border-radius : 25px;</string>
    </property>
    <property name="title">
     <string>Network Test</string>
    </property>
    <widget class="QGroupBox" name="DT_Addr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>481</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Destination Address</string>
     </property>
     <widget class="QPushButton" name="DA_Start">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>25</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DA_lE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="DA_Stop">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>25</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>120</y>
       <width>161</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
     </property>
     <property name="title">
      <string>Keypad</string>
     </property>
     <widget class="QPushButton" name="key_back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>‚Üê</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_0">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="key_point">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:pressed{
background-color: rgb(223, 223, 223);
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QListWidget" name="Output">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>331</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
font-size : 12px;
background-color : white;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 16pt &quot;Agency FB&quot;;
border : 2px solid;
border-radius : 10;

}

QPushButton:pressed{

background-color: rgb(223, 223, 223);

}</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <zorder>Title</zorder>
   <zorder>Network_group</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>IP_BOX</zorder>
   <zorder>pushButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
